### PR TITLE
Add translations for Persistent Volumes for Bahasa Indonesia

### PR DESCRIPTION
**Pull request by evanap**
This PR addresses #13929

### CODE REVIEW

File: content/id/docs/concepts/storage/persistent-volumes.md [hunk 1]
```diff
@@ -0,0 +1,702 @@
...
+reviewers:
+- jsafrane
+- saad-ali
+- thockin
+- msau42
+title: Persistent Volume
+feature:
+  title: Storage orchestration
+  description: >
+    Automatically mount the storage system of your choice, whether from local storage, a public cloud provider such as <a href="https://cloud.google.com/storage/">GCP</a> or <a href="https://aws.amazon.com/products/storage/">AWS</a>, or a network storage system such as NFS, iSCSI, Gluster, Ceph, Cinder, or Flocker.
```
Comment (sftim) [inline content/id/docs/concepts/storage/persistent-volumes.md]: Untranslated?

Comment (irvifa) [reply content/id/docs/concepts/storage/persistent-volumes.md]: Left over english, please translate this

File: content/id/docs/concepts/storage/persistent-volumes.md [hunk 2]
```diff
@@ -0,0 +1,702 @@
...
+{{% /capture %}}
+
+
+{{% capture body %}}
+
+## Pengenalan
+
+Mengelola penyimpanan adalah hal yang berbeda dengan mengelola komputasi. Sub-sistem `PersistentVolume` menyediakan API untuk para pengguna dan administrator yang mengabstraksi detail-detail tentang bagaimana penyimpanan disediakan dari bagaimana penyimpanan dikonsumsi. Untuk melakukan ini, kami mengenalkan dua sumber daya API baru:  `PersistentVolume` dan `PersistentVolumeClaim`.
+
+Sebuah `PersistentVolume` (PV) adalah suatu bagian dari penyimpanan pada kluster yang telah disediakan oleh seorang administrator. PV merupakan sebuah sumber daya pada kluster sama halnya dengan _node_ yang juga merupakan sumber daya kluster. PV adalah _volume plugin_ seperti _Volumes_, tetapi memiliki siklus hidup yang independen dari _pod_ individual yang menggunakan PV tersebut. Objek API ini menangkap detail-detail implementasi dari penyimpanan, seperti NFS, iSCSI, atau sistem penyimpanan yang spesifik pada penyedia layanan _cloud_.
```
Comment (irvifa) [inline content/id/docs/concepts/storage/persistent-volumes.md]: please add comma (,) after sama halnya

File: content/id/docs/concepts/storage/persistent-volumes.md [hunk 3]
```diff
@@ -0,0 +1,702 @@
...
+
+Sebuah `PersistentVolumeClaim` (PVC) merupakan permintaan penyimpanan oleh pengguna. PVC mirip dengan sebuah _pod_. _Pod_ mengonsumsi sumber daya _node_ dan PVC mengonsumsi sumber daya PV. _Pods_ dapat meminta taraf-taraf spesifik dari sumber daya (CPU dan Memory).  Klaim dapat meminta ukuran dan mode akses yang spesifik (seperti, dapat dipasang sekali sebagai _read/write_ atau lain kali sebagai _read-only_).
+
+While `PersistentVolumeClaims` mengizinkan pengguna untuk mengkonsumsi sumber daya abstrak
```
Comment (irvifa) [inline content/id/docs/concepts/storage/persistent-volumes.md]: Left over While? Please translate it

File: content/id/docs/concepts/storage/persistent-volumes.md [hunk 4]
```diff
@@ -0,0 +1,702 @@
...
+  - Jika pengguna tidak menyediakan nama _storage class_, biarkan
+    kolom `persistentVolumeClaim.storageClassName` kosong.
+    - Hal ini kakan membuat sebuah PV disediakan secara otomatis untuk pengguna dengan
+      StorageClass standar pada kluster.  Banyak lingkungan kluster memiliki
+      StorageClass standar yang sudah terpasang, atau administrator dapat membuat
+      StorageClass standar sendiri.
+- Dalam pembuatan, perhatikan PVC yang tidak kunjung terikat setelah beberapa lama
+  dan beritahukan hal ini pada pengguna, karena hal ini dapat mengindikasikan kluster tidak
+  memiliki dukungan storage dinamis (di mana pengguna harus membuat PV yang sesuai)
+  atau kluster tidak memiliki sistem storage (di mana penggun tidak dapat membuat
```
Comment (irvifa) [inline content/id/docs/concepts/storage/persistent-volumes.md]: please make it consistent whether using _storage_ or penyimpanan

File: content/id/docs/concepts/storage/persistent-volumes.md [hunk 5]
```diff
@@ -0,0 +1,702 @@
...
+
+{{< note >}}
+Hanya volume yang disediakan secara statis yang didukung untuk rilis alfa. Administrator harus memperhatikan nilai-nilai tersebut ketika mengerjakan perangkat-perangkat _raw block_.
+{{< /note >}}
+
+## _Volume Snapshot_ dan Dukungan Pemulihan Volume dari _Snapshot_
+
+{{< feature-state for_k8s_version="v1.12" state="alpha" >}}
+
+Fitur _volume snapshot_ ditambahkan hanya untuk mendukung CSI Volume Plugins. Untuk lebih detail, lihat [_volume snapshots_](/docs/concepts/storage/volume-snapshots/).
```
Comment (irvifa) [inline content/id/docs/concepts/storage/persistent-volumes.md]: make CSI Volume Plugins italicized

**Comment by irvifa** [comment]
@evanap It's already long since you submitted this, do you have any progress on this one? Thanks..

**Comment by evanap** [comment]
> @evanap It's already long since you submitted this, do you have any progress on this one? Thanks..

Hi! Sorry for the late update. I've updated the translation with the requested changes.

File: content/id/docs/concepts/storage/persistent-volumes.md (hunk 2 above)
Comment (evanap) [reply content/id/docs/concepts/storage/persistent-volumes.md]: after sama halnya or after dengan node?

File: content/id/docs/concepts/storage/persistent-volumes.md [hunk 6]
```diff
@@ -0,0 +1,697 @@
...
+
+Tergantung metode instalasi, sebuah StorageClass dari setelan standar dapat dibuat
+ke kluster Kubernetes oleh _addon manager_ pada saat instalasi.
+
+Ketika sebuah PVC menspesifikasi sebuah `selector` selain meminta `StorageClass`,
+kebutuhan tersebut akan digabungkan dengan operasi AND bersama: hanya PV dari kelas yang diminta dan dengan
+label yang diminta yang dapat terikat ke PVC.
+
+{{< note >}}
+Saat ini, sebuah PVC dengan `selector` yang tak kosong tidak dapat memiliki PV yang disediakan secara dinmais untuknya.
```
Comment (girikuncoro) [inline content/id/docs/concepts/storage/persistent-volumes.md]: typo:
```suggestion
Saat ini, sebuah PVC dengan `selector` yang tak kosong tidak dapat memiliki PV yang disediakan secara dinamis untuknya.
```

File: content/id/docs/concepts/storage/persistent-volumes.md (hunk 4 above)
Comment (evanap) [reply content/id/docs/concepts/storage/persistent-volumes.md]: okay! resolved in https://github.com/kubernetes/website/pull/14532/commits/9292f151c434f101706b1290e2ecb3e04630b3e5

File: content/id/docs/concepts/storage/persistent-volumes.md (hunk 3 above)
Comment (evanap) [reply content/id/docs/concepts/storage/persistent-volumes.md]: Okay! Resolved in https://github.com/kubernetes/website/pull/14532/commits/d83e05dbfdbf54847b37ef7db85d8b9bd3bfaffe

File: content/id/docs/concepts/storage/persistent-volumes.md [hunk 7]
```diff
@@ -0,0 +1,702 @@
...
+[kube-apiserver](/docs/admin/kube-apiserver/).
+
+### Pengikatan
+
+Seorang pengguna membuat, atau telah membuat (dalam kasus penyediaan dinamis), sebuah `PersistentVolumeClaim` dengan jumlah storage spesifik yang diminta dan dengan mode akses tertentu. Sebuah _control loop_ pada _master_ akan melihat adanya PVC baru, mencari PV yang cocok (jika memungkinkan), dan mengikat PVC dengan PV tersebut. Jika sebuah PV disediakan secara dinamis untuk sebuah PVC baru, _loop_ tersebut akan selalu mengikat PV tersebut pada PVC yang baru dibuat itu. Jika tidak, pengguna akan selalu mendapatkan setidaknya apa yang dimintanya, tetapi volume tersebut mungkin lebih dari apa yang diminta sebelumnya. Setelah terikat, ikatan `PersistentVolumeClaim` bersifat eksklusif, terlepas dari bagaimana caranya mereka bisa terikat. Sebuah ikatan PVC ke PV merupakan _one-to-one mapping_.
```
[the remark this answers is not in the document]
Comment (evanap) [reply content/id/docs/concepts/storage/persistent-volumes.md]: Thanks! Resolved in https://github.com/kubernetes/website/pull/14532/commits/d83e05dbfdbf54847b37ef7db85d8b9bd3bfaffe

File: content/id/docs/concepts/storage/persistent-volumes.md [hunk 8]
```diff
@@ -0,0 +1,702 @@
+---
+reviewers:
+- jsafrane
```
[the remark this answers is not in the document]
Comment (evanap) [reply content/id/docs/concepts/storage/persistent-volumes.md]: Resolved in https://github.com/kubernetes/website/pull/14532/commits/d83e05dbfdbf54847b37ef7db85d8b9bd3bfaffe

File: content/id/docs/concepts/storage/persistent-volumes.md (hunk 1 above)
Comment (evanap) [reply content/id/docs/concepts/storage/persistent-volumes.md]: Sorry for that! Resolved in https://github.com/kubernetes/website/pull/14532/commits/d83e05dbfdbf54847b37ef7db85d8b9bd3bfaffe

File: content/id/docs/concepts/storage/persistent-volumes.md [hunk 9]
```diff
@@ -0,0 +1,702 @@
...
+Dokumen ini menjelaskan kondisi terkini dari `PersistentVolumes` pada Kubernetes. Disarankan telah memiliki familiaritas dengan [volume](/docs/concepts/storage/volumes/).
+
+{{% /capture %}}
+
+
+{{% capture body %}}
+
+## Pengenalan
+
+Mengelola penyimpanan adalah hal yang berbeda dengan mengelola komputasi. Sub-sistem `PersistentVolume` menyediakan API untuk para pengguna dan administrator yang mengabstraksi detail-detail tentang bagaimana penyimpanan disediakan dari bagaimana penyimpanan dikonsumsi. Untuk melakukan ini, kami mengenalkan dua sumber daya API baru:  `PersistentVolume` dan `PersistentVolumeClaim`.
```
[the remark this answers is not in the document]
Comment (evanap) [reply content/id/docs/concepts/storage/persistent-volumes.md]: Okay. Resolved in https://github.com/kubernetes/website/pull/14532/commits/d83e05dbfdbf54847b37ef7db85d8b9bd3bfaffe

File: content/id/docs/concepts/storage/persistent-volumes.md (hunk 5 above)
Comment (evanap) [reply content/id/docs/concepts/storage/persistent-volumes.md]: Resolved in https://github.com/kubernetes/website/pull/14532/commits/d83e05dbfdbf54847b37ef7db85d8b9bd3bfaffe

**Comment by girikuncoro** [comment]
Thanks a lot for your contribution, great work! 🎉  If you are interested and have spare time, you can pick up other pages as well 😄 

/lgtm
/approve

**Comment by evanap** [comment]
> Thanks a lot for your contribution, great work! 🎉 If you are interested and have spare time, you can pick up other pages as well 😄
> 
> /lgtm
> /approve

@girikuncoro Sure! Would love to contribute more to the project!

**Pull request merged**